pages/index.tsx

```diff
@@ -38,3 +42,28 @@ export default function HomePage() {
     </Layout>
   );
 }
+
+export const getServerSideProps: GetServerSideProps = async (_context) => {
+  try {
+    const response = await fetch(`${API_URL}/api/count/packages/`);
+    //to prevent the case where site crashes due to error response
+    if (!response || response.status!==200) {
+      return {
+        props: {
+          packageCount: "all"
+        }
+      }
+    }
```

Comment:
I think we don't need this, if it errors, it will simply not have a packageCount